src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,61 @@
+
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final int ACTION_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+    private static final String SPLIT_INDEX = ",";
+    private static final String BUY_INDEX = "buy";
```

Comment:
```suggestion
    private static final String CSV_SEPARATOR = ",";
    private static final String BUY = "buy";
```

Reply:
Another reviewer said that i need to write this and you are saying to delete this, what I need to do?